app/components/log_summary_component.html.erb

```diff
@@ -3,11 +3,11 @@
     <div class="govuk-grid-column-one-half">
       <header class="app-log-summary__header">
         <h2 class="app-log-summary__title">
-          <%= govuk_link_to lettings_log_path(log) do %>
+          <%= govuk_link_to log.is_a?(LettingsLog) ? lettings_log_path(log) : sales_log_path(log) do %>
```

Comment:
I'd look at adding a `lettings?` method to the base class that defaults to false. Then you can add one to `LettingsLog` that is true:

```suggestion
          <%= govuk_link_to log.lettings? ? lettings_log_path(log) : sales_log_path(log) do %>
```